src/works/index.ts

```diff
@@ -69,6 +78,15 @@ export const getWork = async ({
     },
     publishedAt: getWorkPublishDate(workPage),
     updatedAt: getWorkUpdateDate(workPage),
+    chapter:
+      chapterId && totalChapters !== 1
```

Comment:
I've been thinking about this a bit, and I think that the `&& totalChapters !== 1` isn't necessary. 

AO3 has three chapter configurations:
- #/? - these might have a chapterId provided by AO3 if there's more than one chapter, but won't if there's only one chapter. 
- 1/1 - This is a single chapter fic aka one shot. These are not provided a chapterId by AO3.
- #/# - where #> 1 in both cases. All chapters will have a chapterID.

Since AO3 doesn't provide chapterIDs if there's only one chapter, I think we can simplify this to just `chapterId` what do you think?

Reply:
That will work. The `totalChapters` check was the way I was originally checking if the object should be `null` or not. I'll update it.

Reply:
Actually, I remembered why the `totalChapters` check was still there. It's so a chapter summary can still be returned for a multi-chapter fic, even if no `chapterId` is passed.